src/docs/product/accounts/quotas/index.mdx

```diff
@@ -35,27 +35,29 @@ Sentry completes a thorough evaluation of each event to determine if it counts t
 
 3. **Quota availability**
 
-   Events that exceed your quota are not accepted. Once your event volume is approaching or has exceeded the quota, teammates with the "owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. To add to your quota or review what happens when you exceed it, see [Increasing Quotas](/product/accounts/quotas/#increasing-quotas).
+   Events that exceed your quota are not accepted. Once your event volume is approaching or has exceeded the quota, teammates with the "owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. To add to your quota or review what happens when you exceed it, see [Increasing Quotas](#increasing-quotas).
 
 4. **Event repetition**
 
-- If you have intervened to Delete and Discard an issue, then _future_ events with the same fingerprint do not count toward your quota.
-- If the previous event was resolved, this event counts toward your quota because it may represent a regression in your code.
-- If you have intervened to ignore alerts about events with the same fingerprint, this event counts toward your quota because the event is still occurring. For more information, see [Inbound Filters](/product/accounts/quotas/#inbound-data-filters).
+    Event repeition only applies to error events, not transactions or attachments. The following rules apply for event repeition and your quota:
+
+    - If you have intervened to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.
+    - If the issue was previously was resolved, this error event counts toward your quota because it may represent a regression in your code.
+    - If you have intervened to ignore alerts about error events with the same fingerprint, this event counts toward your quota because the event is still occurring. For more information, see [Inbound Filters](#inbound-data-filters).
```

Comment:
```suggestion
    - If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).
```